docs/source/install/all_in_one.rst

```diff
@@ -0,0 +1,136 @@
+All-in-one Installation
+=======================
+|st2| provides an all-in-one installer aimed at assisting users with the initial setup and configuration. The installer comes pre-bundled in a number of different provisioning options for convenience, or can also be manually deployed and installed manually on a server.
+
+Pre-Requsites
+-------------
+Before getting started, it is necessary to do a bit of pre-planning to integrate |st2| into an environment. Below is a checklist of items to prepare before running the installer.
+
+#. A server. See below for the list of pre-bundled options, or bring your own.
+#. Hostname of new server setup with DNS.
+#. Admin Password.
+#. Administrative SSH Public/Private Keys *(Optional)*
+#. SSL Certificates *(Optional)*
+#. Chat service connection credentials *(Optional)*
+
+The initial setup is designed to configure itself to a freshly deployed server with no additional services running. If you do not have or do not wish to provide SSL and SSH keys during the initial setup, |st2| will generate random new keys automatically.
+
+Sizing the Server
+-----------------
+A standard all-in-one installation has all of the various components of |st2|, as well as the supporting infrastructure components. While the system can operate with less equipped servers, these are recommended for the best experience while testing or deploying |st2|.
+
+Testing
+~~~~~~~
+* Single or Dual CPU system
+* At least 4GB of RAM
```

Comment:
4Gig looks a lot for me too.

Don't really know this Installer requirements, but I successfully installed production with all st2 components, Chatops with Hubot and Ansible, custom packs on `t2.small` with 2 GiB RAM  [~20$/month](https://aws.amazon.com/ec2/pricing/) and it looks pretty well as minimum viable configuration: 
![aws-st2-usage](https://cloud.githubusercontent.com/assets/1533818/9196503/9134d202-4034-11e5-91ec-700cfce17c9d.gif)

My thinking about high requirements - is not to make it barrier. But you know better, just my 0.01c.